int32 centerX
int32 centerY
bool isDetected